frontend/src/app/patients/ManagePatients.tsx

```diff
@@ -128,4 +194,62 @@ const ManagePatients = ({ activeFacilityId, canEditUser }: Props) => {
   );
 };
 
+const ManagePatients = (
+  props: Omit<Props, InjectedQueryWrapperProps | "pageCount">
+) => {
+  const activeFacilityId = useSelector(
+    (state) => (state as any).facility.id as string
+  );
+
+  const {
+    data: totalPatients,
+    loading,
+    error,
+    //refetch: refetchCount,
```

Comment:
Extraneous comment unless this is intended to be a reminder for future functionality

Reply:
Actually this should have had a TODO in front of it. 😸  It was to remind me to refresh the total patient count. Done.